### PR TITLE
update Related Projects section

### PR DESCRIPTION
**Pull request by mforets**

### CODE REVIEW

File: docs/src/about.md [hunk 1]
```diff
@@ -91,7 +91,18 @@ Note that this also runs all doctests which will take some time.
...
+and systems definitions for reachability problems within [JuliaReach](https://github.com/JuliaReach/).
+
+Below we list more related projects.
+
+|Package name|Description|
+|----|------|
+|[HybridSystems.jl](https://github.com/blegat/HybridSystems.jl)|Hybrid Systems definitions in Julia.|
+|[LTISystems.jl](https://github.com/JuliaSystems/LTISystems.jl)|Julia package for representing linear, time-invariant system models and operations defined on them.|
+|[ControlToolbox.jl](https://github.com/JuliaSystems/ControlToolbox.jl)|Analysis and design tools for control systems.|
+|[ControlToolbox.jl](https://github.com/ronisbr/ControlToolbox.jl)|A Control Toolbox for Julia language.|
```
Comment (schillic) [inline docs/src/about.md]: Why the duplicate?

File: docs/src/about.md [hunk 2]
```diff
@@ -91,7 +91,18 @@ Note that this also runs all doctests which will take some time.
...
+|[DynamicalSystemsBase.jl](https://github.com/JuliaDynamics/DynamicalSystemsBase.jl)|Definitions of core system and data types used in in the ecosystem of DynamicalSystems.jl.|
```
Comment (schillic) [inline docs/src/about.md]: "in in"

File: docs/src/about.md [hunk 3]
```diff
@@ -91,7 +91,18 @@ Note that this also runs all doctests which will take some time.
...
-- [Hybrid Systems](https://github.com/blegat/HybridSystems.jl)
+This package originated from [Hybrid Systems](https://github.com/blegat/HybridSystems.jl)
+and systems definitions for reachability problems within [JuliaReach](https://github.com/JuliaReach/).
+
+Below we list more related projects.
+
+|Package name|Description|
+|----|------|
+|[HybridSystems.jl](https://github.com/blegat/HybridSystems.jl)|Hybrid Systems definitions in Julia.|
+|[LTISystems.jl](https://github.com/JuliaSystems/LTISystems.jl)|Julia package for representing linear, time-invariant system models and operations defined on them.|
```
Comment (schillic) [inline docs/src/about.md]: I think no comma between "linear time-invariant"

**Comment by blegat** [comment]
We could also add https://github.com/JuliaControl/ControlSystems.jl

File: docs/src/about.md (hunk 1 above)
Comment (mforets) [reply docs/src/about.md]: seems like the fork has been more recently updated.

Comment (schillic) [reply docs/src/about.md]: Then maybe add "Fork" or something to make clear which one is the fork?

Comment (schillic) [reply docs/src/about.md]: 👍 for your solution

**Pull request merged**